sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/spark/diagnostics/SimpleDiagnosticsSlf4jLogger.scala

```diff
@@ -7,9 +7,8 @@ import com.azure.cosmos.implementation.spark.OperationContext
 import com.azure.cosmos.implementation.{HttpConstants, OperationType, RxDocumentServiceRequest, RxDocumentServiceResponse}
 
 // scalastyle:off multiple.string.literals
-
 private[spark] class SimpleDiagnosticsSlf4jLogger(classType: Class[_])
-  extends DefaultMinimalSlf4jLogger(classType: Class[_]) {
+  extends FeedDiagnosticsSlf4jLogger(classType: Class[_]) {
```

Comment:
is this change purposely?

Reply:
Yes - want the feed diagnostics to also show up in Simple Diagnostics